tests/unit/mongodb_test_charm/actions.yaml

```diff
@@ -0,0 +1,54 @@
+get-primary:
```

Comment:
is the mongodb test charm needed? I imagine it might be annoying to update this "mongo test charm" as the mongodb/mongos/k8s/vm charms change 

but also I think it might be the only solution/the cleanest? So prob good :) but still curious what you think

Reply:
For now : I don't have a clean solution, this is mainly here so that I can have some unit testing and some deployment that I can test.

My idea is that this test charm would be very basic (I would say one mongodb and one mongos) and I use it for unit testing and that's all.

I hope to find a better solution but for now I don't so I'll keep it that way :)